tensorboard/uploader/uploader_test.py

```diff
@@ -1337,6 +1342,8 @@ def test_break_at_scalar_point_boundary(self):
                 total_points_in_result += 1
             self.assertLessEqual(request.ByteSize(), 1024)
         self.assertEqual(total_points_in_result, point_count)
+        with self.subTest("Scalar report count correct."):
+            self.assertEqual(tracker._stats.num_scalars, point_count)
```

Comment:
Hmm… okay. I’m not thrilled about poking into the implementation details
of `UploadTracker`, but it also looks like the current API doesn’t
expose a way to get this information directly, so I’ll allow it.